src/app/static/src/app/components/modelCalibrate/ModelCalibrate.vue

```diff
@@ -106,7 +114,10 @@
         methods: {
             update: mapMutationByName(namespace, ModelCalibrateMutation.Update),
             calibrate: mapActionByName(namespace,"calibrate"),
-            fetchOptions: mapActionByName(namespace, "fetchModelCalibrateOptions")
+            fetchOptions: mapActionByName(namespace, "fetchModelCalibrateOptions"),
+            submitForm() {
+                (this.$refs.form as any).submit()
```

Comment:
Yeah, I forgot about this separate button business when I blithely put this down as a one pointer! We do want submission to go via the form, because it reshapes DynamicFormMeta into DynamicFormOptions for us. This seems a bit ugly but can't think of a simple alternative.

Reply:
Yep, I think this is the way :+1: